tensorflow/python/eager/context.py

```diff
@@ -1244,6 +1244,24 @@ def invoking_op_callbacks(self):
   def invoking_op_callbacks(self, value):
     self._thread_local_data.invoking_op_callbacks = value
 
+  # Parse PhysicalDevice from the device string
+  # if the device string contains subdevice type string, then parse it.
```

Comment:
Capitalize "If" here and below. Also end previous sentence: "Parse PhysicalDevice from the device string."

Also can you add examples for the two cases in the comment?

Reply:
Done.